orttraining/orttraining/core/graph/training_op_defs.cc

```diff
@@ -1118,6 +1118,46 @@ void RegisterTrainingOpSchemas() {
           {"float"},
           "Constrain learning rate to float");
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(SGDOptimizerV2)
+      .SetDomain(kMSDomain)
+      .SinceVersion(1)
+      .Input(0, "lr", "The learning rate.", "T1")
+      .Input(1, "weights", "Sequence of weights to optimize.", "S_WEIGHT")
+      .Input(2, "gradients", "Sequence of gradients computed in this iteration.", "S_GRAD")
+      .Input(3, "update_signal",
+             "This signal indicates if weight updates are skipped, applicable to gradient infinity check"
+             " in mixed precision training. ",
+             "T_BOOL", OpSchema::Optional)
+      .Output(0, "updated_flag", "Whether gradient is applied or not.", "T_BOOL")
```

Comment:
nit: rename to update_completed to make it more intuitive?

Reply:
done